sdk/core/src/main/java/com/klaviyo/coresdk/networking/KlaviyoEvent.kt

```diff
@@ -1,5 +1,10 @@
 package com.klaviyo.coresdk.networking
 
+/**
+ * Events recognized by Klaviyo
```

Comment:
This might be misleading since we can have custom events. Maybe we could call these common events tracked in Klaviyo?

Reply:
True. I actually have a value in this sealed class for specifying a custom event but the comment isn't highlighting its existence so I'll upload to include a note on that